src/components/layout/container.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { ReactNode } from 'react'
+import * as Styled from './styles'
+
+interface Props {
+  children: ReactNode
+}
+
+const Container: React.FC<Props> = ({ children }: Props) => {
```

Comment:
This seems completely redundant. Instead of declaring and using `Props` type that only contains unchanged `children` just use `FC` without generic type.

```suggestion
const Container: React.FC = ({ children }) => {
```

Reply:
Yep I just copied it from the blog.

Reply:
This seem like it's not working with eslint. 😢 
<img width="542" alt="Screenshot 2020-11-12 at 16 06 40" src="https://user-images.githubusercontent.com/3129012/98957078-11471c00-2501-11eb-9e87-319b51b81877.png">